modules/ROOT/pages/configuring-dev-env.adoc

```diff
@@ -53,3 +54,191 @@ include::{partialsdir}/attributes.adoc[]
+
+ * The latest version of http://openjdk.java.net/install/index.html[OpenJDK]
+
+*Using a Published SDK*
+
+_jcenter()_
+
+To use a published version of the SDK you have to add jcenter to the build.grade file in your project's root directory
+
+[source, groovy]
+----
+allprojects {
+    repositories {
+        jcenter() // <-- Add This line
+        google()
+    }
+}
+----
+
+*Reference the Dependencies*
+
+In the build.gradle file of your app you can begin adding dependencies provided by the SDK.
+
+[source,groovy]
+----
+dependencies {
+    ...
+    implementation 'org.aerogear:android-push:[version]'
+    implementation 'org.aerogear:android-auth:[version]'
+    implementation 'org.aerogear:android-security:[version]'
+}
+----
+
+*Additional Dependencies*
+
+For push you also need to add a dependency to the Firebase SDK:
+
+[source,groovy]
+----
+dependencies {
+    ...
+    implementation "com.google.firebase:firebase-messaging:[version]"
+}
+----
+
+A full list of releases can be viewed https://github.com/aerogear/aerogear-android-sdk/releases[here]
+
+*Certificate Pinning*
+
+To implement certificate pinning in the individual SDKs, see link:certificate-pinning.html[the certificate pinning guide].
+
+****
+
+[role="secondary"]
+.iOS
+
+****
+
+*Prerequisites*
+
+To use the AeroGear SDK for iOS, you will need the following installed on your development machine:
+
+* iOS 9 or later
+* Xcode 9 or later
+* link:https://cocoapods.org/[CocoaPods]
+
+To get started with the AeroGear iOS SDK, you can set up the SDK in your existing or new project. The SDK is written in Swift.
+
+*CocoaPods*
+
+. The AeroGear SDK for iOS is available using the CocoaPods package manager.
+If you have not installed CocoaPods, install it by running the command:
++
+[source,bash]
+----
+gem install cocoapods
+pod setup
+----
++
+. In your project directory (the directory where your `.xcodeproj` file is), create a plain text file named `Podfile` (without any file extension) and add the lines below. Replace `YourTarget` with your actual target name.
++
+[source,ruby]
+----
+source 'https://github.com/CocoaPods/Specs.git'
++
+platform :ios, '9.0'
+use_frameworks!
+
+target :'YourTarget' do
+    ## Require one of the AeroGear SDK's here
+    pod 'AGSPush'
+    pod 'AGSAuth'
+end
+----
++
+. Then run the following command:
++
+[source,bash]
+----
+pod install
+----
++
+. Open up `.xcworkspace` with Xcode and start using the SDK.
++
+> Do not use `.xcodeproj`. If you open up a project file instead of a workspace, dependencies will not be correctly configured.
++
+. Import SDK you want to use in the application
+For example:
+
+[source,swift]
+----
+   import AGSAuth
+----
+
+****
+
+[role="secondary"]
+.Cordova
+
+****
+
+*Prerequisites*
+
+Before installing the SDK you will need in your system, at least:
+
+* link:https://nodejs.org/[Node.js] runtime and NPM
+* link:https://cordova.apache.org/docs/en/latest/guide/cli/#installing-the-cordova-cli[Cordova CLI]
```

Comment:
This is tricky actually. For showcase you do not need anything as we using ionic that wraps cordova and is installed locally. If users start their own app they may need cordova 8.0.0! no other version is supported. Giving you free choice how this could be documented.